core/async/src/test_loop.rs

```diff
@@ -239,6 +241,18 @@ impl<Event: Debug + Send + 'static> TestLoopBuilder<Event> {
     }
 }
 
+impl<Event: Debug + Send + 'static> TestLoopBuilder<Event> {
+    pub fn register_actor<A>(&self, actor: A) -> TestLoopSender<A>
```

Comment:
can this be inside the previous "impl" block for TestLoopBuilder?

Reply:
Yes, it can, but coming up I have an implementation for register_actor_for_index (or something similar) which needs `TestLoopBuilder<(usize, Event)>`. That said, I can push it to the same impl block.